### PR TITLE
Feat/unify types android ios

### PR DESCRIPTION
**Pull request by GuillaumeEgret**
Update the code of ios-test to use the same types for iOS as Android.
Compute the value of CPU usage in percentage instead of CPU cycles

### CODE REVIEW

File: packages/ios-poc/.gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 report
 *.trace
 report.json
+.DS_Store
+*.xml
+*.json
```
Comment (legkikh) [inline packages/ios-poc/.gitignore:6]: We would not able to push changes in `package.json`, etc?

File: packages/ios-poc/src/writeReport.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    }, new Map<number, number[]>());
+  };
+
+  const options = {
+    attributeNamePrefix: "",
+    ignoreAttributes: false,
+    parseAttributeValue: true,
+    textNodeName: "value",
+    updateTag(tagName: string, jPath: string, attrs: { [x: string]: string | number }) {
+      if (tagName === "trace-query-result") return "result";
```
Comment (niebo26) [inline packages/ios-poc/src/writeReport.ts:58]: Maybe a Switch instead?

File: packages/types/index.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export interface TestCaseResult {
   score?: number;
   status: TestCaseResultStatus;
   iterations: TestCaseIterationResult[];
-  type?: undefined;
+  type?: undefined | "IOS_EXPERIMENTAL";
```
Comment (niebo26) [inline packages/types/index.ts:37]: [To Investigate / Not mandatory] Replace undefined with a more explicit "ANDROID" string type?

File: packages/ios-poc/src/writeReport.ts [hunk 4]
```diff
@@ -46,6 +64,9 @@ export const writeReport = (inputFileName: string, outputFileName: string) => {
   const parser = new XMLParser(options);
   const jsonObject: Result = parser.parse(xml);
 
+  const fistSampleTime: number = jsonObject.result.node.row[0].sampleTime.value / 1_000_000;
```
Comment (niebo26) [inline packages/ios-poc/src/writeReport.ts:67]: Explicit 1000000 with a constant name

**Review by niebo26 — APPROVED** [review]

**Pull request merged**